src/constants.ts

```diff
@@ -11,8 +11,8 @@ export const HEXADECIMAL_KEY_LENGTH = 128 as const;
 export const BASE_64_REGEX =
   /^(?:[A-Za-z0-9+/]{4})*(?:[A-Za-z0-9+/]{3}=|[A-Za-z0-9+/]{2}==)?$/u;
 
-export const MIN_BIP_44_DEPTH = 0 as const;
-export const MAX_BIP_44_DEPTH = 5 as const;
+export const MIN_BIP_44_DEPTH = 0;
+export const MAX_BIP_44_DEPTH = 5;
```

Comment:
Why remove the `as const`? Were they redundant?

Reply:
Yeah, TypeScript infers the types of these consts as `0` and `5` respectively without the `as const` assertion too.